### DISCUSSION TITLE
Add network/chain on-the-fly

### DISCUSSION THREAD

**Discussion by Exzender** [WalletConnect/walletconnect-monorepo#3781] [Developer Support]
I want to add another blockchain to web3modal list after modal was created with createWeb3Modal.
Use-case: user adds custom network / chain on-the-fly.

Is it possible to change the configuration of web3modal after creation?
Or maybe there is a way to destroy previously created web3modal, change config and recreate it again with new call to createWeb3Modal ?

**Comment by glitch-txs** [comment]
What's the use case? how will the user interact with that chain? what type of dapp would use it?
